kratos/solving_strategies/strategies/residualbased_newton_raphson_strategy.h

```diff
@@ -259,6 +259,7 @@ class ResidualBasedNewtonRaphsonStrategy
      
     virtual ~ResidualBasedNewtonRaphsonStrategy()
     {
+        Clear();
```

Comment:
Hello,

i have two comments on this
1 - the behaviour of the trilinos app should be as close as possible to the one of the serial one. 
2 - i think that calling Clear on destruction makes sense... why do you differ?